driver/normalizer/normalizer.go

```diff
@@ -1019,13 +1187,26 @@ func (op opMergeGroups) Kinds() nodes.Kind {
 	return nodes.KindObject
 }
 
-// Check tests if the current node is uast:Group and if it contains a uast:FunctionGroup
-// node, it will remove the current node and merge other children into the FunctionGroup.
+// Check tests if a current node is uast:Group and uast:FuncGroup and contains group of
+// another kind. It will remove the second group and merge children into current one.
+// uast:FuncGroup is preferred.
```

Comment:
Maybe link to: https://github.com/bblfsh/sdk/issues/361 so we don't forget to update once we've a better solution for this.